server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -95,23 +94,18 @@ protected VolumeManagerImpl(Map<String,Volume> volumes, Volume defaultVolume,
     this.hadoopConf = hadoopConf;
   }
 
-  private void invertVolumesByFileSystem(Map<String,Volume> forward,
-      Multimap<URI,Volume> inverted) {
-    for (Volume volume : forward.values()) {
-      inverted.put(volume.getFileSystem().getUri(), volume);
-    }
+  private Multimap<URI,Volume> invertVolumesByFileSystem(Map<String,Volume> forward) {
+    Multimap<URI,Volume> inverted = HashMultimap.create();
+    forward.values().forEach(volume -> inverted.put(volume.getFileSystem().getUri(), volume));
+    return inverted;
   }
 
-  public static org.apache.accumulo.server.fs.VolumeManager getLocal(String localBasePath)
-      throws IOException {
+  // for testing only
+  public static VolumeManager getLocalForTesting(String localBasePath) throws IOException {
```

Comment:
I don't like that annotation, because:

1. it can create an unnecessary Guava dependency,
2. I don't like checking to see if we already have Guava on that module, and
3. the annotation provides no additional value over a comment since there are no tools for which the annotation would matter

The method name makes it clear. The comment is bonus. An annotation wouldn't do anything other than add an import and possibly a new direct dependency if Guava wasn't already in the pom.

Reply:
This makes me think we should just create our own annotation for VisibleForTesting since its not used for anything else other than a label.

Reply:
Maybe. That would eliminate the issue of bringing in a dependency when not needed, but it still wouldn't matter whether it's an annotation or a comment if we don't have any tooling for verification.